.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-  presets: ['next/babel', '@babel/typescript', 'minify'],
+  presets: [
+    ['next/babel', { 'transform-runtime': { corejs: false } }],
```

Comment:
Thanks @ykzts.  Sorry I haven't had a chance to read up on this just yet. Would you consider this a breaking change? Will it affect older browsers for example on the current version of next? 

I will look at getting this released this evening after work.

Reply:
The only difference is whether or not core-js is used. It does not affect older browsers or previous versions of Next.js.